src/CoreWCF.MSMQ/src/CoreWCF/Channels/MsmqNetcoreTransport.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO.Pipelines;
+using System.Threading;
+using System.Threading.Tasks;
+using CoreWCF.Configuration;
+using CoreWCF.Queue.Common;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using MSMQ.Messaging;
+using MSMQM = MSMQ.Messaging;
+
+namespace CoreWCF.Channels
+{
+    public class MsmqNetcoreTransport : IQueueTransport, IDisposable
+    {
```

Comment:
No need to have Netcore in the name. Maybe MsmqQueueTransport, basically strip the I and prefix the purpose.